sql/parse/util.go

```diff
@@ -516,3 +516,65 @@ func maybeList(opening, separator, closing rune, list *[]string) parseFunc {
 		}
 	}
 }
+
+// A QualifiedName represents an identifier of type "db_name.table_name"
+type QualifiedName struct {
```

Comment:
is this used outside this package? If not, I would keep it private

Reply:
I wanted to keep it private too, but I use it in `views.go` when calling `readQualifiedIdentifierList`.

Reply:
Done in https://github.com/src-d/go-mysql-server/pull/860/commits/9b8ecfaa7946a9b87e74946736b6566c205f6f04.